portfolio/src/main/webapp/index.html

```diff
@@ -8,14 +8,83 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <h2>Theodora-Mara Pislar</h2>
-      <p>This is my portfolio.</p>
-      <p>Here you will find more things about me.</p>
+      <h1 style = "text-align : center;">My Portfolio</h1>
+      <h2 style = "text-align : center;">Theodora-Mara Pîslar</h2>
+      <p>You can find me here:
+        <ul>
+         <li> <a href=
```

Comment:
https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping
It looks like the styleguide asks for indenting continuations when wrapping by 4 spaces